docs/Project.toml

```diff
@@ -0,0 +1,9 @@
+[deps]
+DataFrames = "a93c6f00-e57d-5684-b7b6-d8193f3e46c0"
+Documenter = "e30172f5-a6a5-5a46-863b-614d45cd2de4"
+Impute = "f7bf1975-0170-51b9-8c5f-a992d46b9575"
+RDatasets = "ce6b1742-4840-55fa-b093-852dadbb1d8b"
+
+[compat]
+DataFrames = ">= 0.16"
```

Comment:
I think it's fine because we don't required DataFrames for the main package and this'll make maintenance easier. I've actually updated the main Project.toml now because it also doesn't need to be that strict with DataFrames not being a direct dependency.

Reply:
I've been thinking this also re:DataFrames
and other package where we depend only on very superficial or core functionality
(like the existance of a `AbstractDataFrame` type).
that is unlikely to change.

I wish we could leave coments in Project.toml files to explain why this is the Case.

Reply:
plus this is only the docs